pkg/pbq/store/noop/store.go

```diff
@@ -0,0 +1,36 @@
+package noop
+
+import (
+	"github.com/numaproj/numaflow/pkg/isb"
+	"github.com/numaproj/numaflow/pkg/pbq/store"
+)
+
+// PBQNoOpStore is a no-op pbq store which does not do any operation but can be safely invoked.
+type PBQNoOpStore struct {
+}
+
+var _ store.Store = (*PBQNoOpStore)(nil)
+
+func NewPBQNoOpStore() (*PBQNoOpStore, error) {
+	return &PBQNoOpStore{}, nil
+}
+
+func (p *PBQNoOpStore) Read(size int64) ([]*isb.Message, bool, error) {
+	return []*isb.Message{}, true, nil
+}
+
+func (p *PBQNoOpStore) Write(msg *isb.Message) error {
+	return nil
+}
+
+func (p *PBQNoOpStore) Close() error {
+	return nil
+}
+
+func (p *PBQNoOpStore) GC() error {
+	return nil
+}
+
+func (p *PBQNoOpStore) IsEmpty() bool {
+	return false
```

Comment:
shouldn't this be `true`?

perhaps we do not require `IsEmpty` to be in the interface. if yes, unexport it.

Reply:
done, removed it from the interface